### ISSUE TITLE
Thanks Page Missing

### ISSUE THREAD

**Issue by mchladek** [chihacknight/chihacknight.org#646]
After making a donation / member fee via PayPal, PayPal directs back to https://chihacknight.org/thanks.html However, that link results in a 404.

**Comment by derekeder** [comment]
@mchladek I don't believe this page ever existed on our website. My guess is its a default setting in PayPal. Could you try and update the settings to have it redirect to https://chihacknight.org/index.html ?

**Comment by EvanHahn** [comment]
Just ran into this myself and looked into fixing it.

I don't know much about PayPal integrations, but [reading their docs][0] makes it look like you can add the following to the form in `donate/index.html`:

```diff
 <form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top" style="border: none;">
   <input type="hidden" name="cmd" value="_s-xclick" />
   <input type="hidden" name="hosted_button_id" value="822LXFNLVXQ5Y" />
+  <input type="hidden" name="return" value="https://chihacknight.org/" />
   <input type="image" src="/images/donate/paypal.png" border="0" style="height: 100px; border: none;" name="submit" title="Donate with PayPal" alt="Donate with PayPal button" />
   <img alt="" border="0" src="https://www.paypal.com/en_US/i/scr/pixel.gif" width="1" height="1" />
 </form>
```

I haven't tested this so it might not work. If it doesn't, [this Stack Overflow answer][1] might also be useful.

[0]: https://developer.paypal.com/api/nvp-soap/paypal-payments-standard/integration-guide/Appx-websitestandard-htmlvariables/
[1]: https://stackoverflow.com/a/7708419

**Comment by mchladek** [comment]
Thanks, @EvanHahn! Your comment prompted me to look into this again 😊 I ended up changing the settings within PayPal, so I think it should redirect to the main page now. Although, I think your solution would work, too, Evan, if for some reason the settings I adjusted within PayPal don't do the trick. But I'll close the issue for now.

**Comment by EvanHahn** [comment]
Just tested this with a $1 donation and it's working great!

**Comment by mchladek** [comment]
Thanks so much, @EvanHahn, for testing and confirming! Very much appreciated!
